src/react-components/home/SocialBar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./SocialBar.scss";
+import { Container } from "../layout/Container";
+import { ReactComponent as Discord } from "../icons/SocialDiscord.svg";
+import { ReactComponent as Twitch } from "../icons/SocialTwitch.svg";
+import { ReactComponent as Twitter } from "../icons/SocialTwitter.svg";
+import { ReactComponent as Vimeo } from "../icons/SocialVimeo.svg";
+import { ReactComponent as Youtube } from "../icons/SocialYoutube.svg";
+export function SocialBar({ mobile }) {
+  return (
+    <Container className={mobile ? styles.mobileSocialBar : styles.socialBarContainer}>
+      <a target="_blank" href="https://discord.com/invite/dFJncWwHun">
```

Comment:
Todo in the future, get these links to be defined in db so HC users can enter their socials in the admin panel.

Reply:
This might be a good one to run by the design team, I didn't consider this an HC feature but maybe it's something HC users would want. Should probably double check that this is wrapped in an `isHmc` conditional for now so it doesn't show up on HC until it is validated and can add in a future PR.